badge-maker/index.d.ts

```diff
@@ -4,6 +4,10 @@ interface Format {
   labelColor?: string
   color?: string
   style?: 'plastic' | 'flat' | 'flat-square' | 'for-the-badge' | 'social'
+  logo?: string
+  logoPosition?: number
+  logoWidth?: number
+  links?: Array<string>
```

Comment:
Adding links is issue https://github.com/badges/shields/issues/4949
The reason this has been sitting around for a while is because `links` is implemented internally as an array for historical reasons, rather than using the `label` and `message` terminology, but we'd really like to add it as `labelLink` and `messageLink` to match the other params.

The conclusion we came to on this (in https://github.com/badges/shields/pull/5542#issuecomment-692956549 ) was

> We should expose the public API `labelLink` and `messageLink` in the package, but its a bigger job than just setting `links = [labelLink, messageLink]` because currently `links[0]` affects the whole badge when `links.length == 1` but only the left side when `links.length == 2`. To make `labelLink` make sense as a param it should only ever affect the label (left) side of the badge but then just making that change in the package is then a BC break for the server so when we call out to `makeBadge()` in the server component we'll need to translate that to
> 
> ```js 
> labelLink = links[0]
> messageLink = links[0]
> ```
> 
> if `links.length == 1` to retain the existing behaviour of `?link` for SaaS users. We'll also need tests for all that.

but the PR was never finished.